var/spack/repos/builtin/packages/bison/package.py

```diff
@@ -31,6 +31,9 @@ class Bison(AutotoolsPackage):
 
     patch('pgi.patch', when='@3.0.4')
 
+    # Intel 14 has immature C11 support
```

Comment:
You could put this comment in a `msg='...'` argument if you want it printed on the command-line.

Reply:
Good idea, I'll update.